### PR TITLE
MS. Fairfax 16: New description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

**Comment by adunning** [comment]
This could have far more detail, but hopefully it's a good enough for now.

File: collections/Fairfax/MS_Fairfax_16.xml [hunk 1]
```diff
@@ -37,49 +36,67 @@
...
+                     <decoNote type="miniature">Fine drawing (retouched) of 16th century, end, fol. 9r. (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 974, pl. XCI)</decoNote>
+                     <decoNote type="rubrication">Rubricated in the hand of the scribe. Occasionally, as on fol. 115r, enclosed in a red parallelogram. Marginal headings and running heads in red ink preceded by blue paraphs.</decoNote>
+                     <decoNote type="flourInit">Flourised initials in blue with red penwork extending up and down the margins to begin new items.</decoNote>
+                     <decoNote type="colInit">Red tipping of many upper case letters mainly at the beginning of lines.</decoNote>
                   </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1900" notBefore="1800">
+                        <p>Light brown split calf over laminated pulpboard, 19th century; repaired 1949.</p>
```
Comment (holfordm) [inline collections/Fairfax/MS_Fairfax_16.xml:73]: is this still accurate? wasn't it rebound recently? (there may be a conservation report worth checking as well)

File: collections/Fairfax/MS_Fairfax_16.xml [hunk 2]
```diff
@@ -92,14 +109,662 @@
...
+                        <note>Full-page illustration of the 'Complaint of Mars and Venus'</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 15r–20v)</locus>
+                        <rubric>Complaynt of Mars and Venus</rubric>
+                        <msItem>
+                           <locus>(fols. 15r–19r)</locus>
+                           <author key="person_100185203">Geoffrey Chaucer</author>
+                           <title key="work_6695">Compleynt of Mars</title>
+                           <rubric>Complaynt of Mars and Venus</rubric>
```
Comment (holfordm) [inline collections/Fairfax/MS_Fairfax_16.xml:172]: rubric repeated? or same as l. 167?

File: collections/Fairfax/MS_Fairfax_16.xml [hunk 3]
```diff
@@ -92,14 +109,662 @@
...
+                        <title>Prose list of heraldic regulations</title>
+                        <rubric>A remembraunz off þe articles perteyning to þe ordyr off rodys</rubric>
+                        <incipit>First ye schal swere to owre souerreyn Lord the kynge</incipit>
+                        <explicit>to kepe with alle youre powere. So help you god and holydome.</explicit>
+                        <note>Added in a secretary bookhand of the late 15th century; in a second hand from fol. 330r.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 330v–332v)</locus>
+                        <author key="person_22158282">John Lydgate</author>
+                        <title type="desc">Verses on the Kings of England</title>
```
Comment (holfordm) [inline collections/Fairfax/MS_Fairfax_16.xml:750]: work_2734

File: collections/Fairfax/MS_Fairfax_16.xml [hunk 4]
```diff
@@ -37,49 +36,67 @@
...
+                  <bindingDesc>
+                     <binding notAfter="1900" notBefore="1800">
+                        <p>Light brown split calf over laminated pulpboard, 19th century; repaired 1949.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" when="1450">1450 (?)</origDate>
+                     <origDate calendar="Gregorian" when="1450">Inscribed, <q>Anno 1450</q> (fol. 1r).</origDate>
```
Comment (holfordm) [inline collections/Fairfax/MS_Fairfax_16.xml:79]: would it be safer to say mid 15th cent. (?1450) or something similar?

File: collections/Fairfax/MS_Fairfax_16.xml [hunk 5]
```diff
@@ -37,49 +36,67 @@
...
                      <origPlace>
                         <country key="place_7002445">English</country>
                      </origPlace>
                   </origin>
+                  <provenance><persName key="person_000000" role="fmo">John Stanley (d. 1469?)</persName> of Hooton, Cheshire: coat of arms incorporated into border and illustration (fol. 14v). Also owned MS. Harley 6163.</provenance>
+                  <provenance>Possible pressmark from the late 15th century, <q>15</q> (fol. ii verso).</provenance>
+                  <provenance>Inscribed, <q>for the care <lb/>
+                        <unclear>anne</unclear> wer <lb/> and in the <gap reason="deleted"/></q> (fol. ii verso); <q><unclear>dame</unclear> Anne <unclear>W<ex>er</ex></unclear> scripsit</q>(fol. 340v).</provenance>
+                  <provenance>Inscribed, <q>John <lb/> dvinsins sola</q> (fol. 12v), 16th century.</provenance>
```
Comment (holfordm) [inline collections/Fairfax/MS_Fairfax_16.xml:88]: dvinsins: sic?

File: collections/Fairfax/MS_Fairfax_16.xml [hunk 6]
```diff
@@ -92,14 +109,662 @@
...
+                        <idno type="part">MS. Fairfax 16, fols. 202–305</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="en" otherLangs="la">Middle English and Latin</textLang>
+                     <msItem>
+                        <locus>(fols. 202r–300r)</locus>
+                        <author key="person_22158282">John Lydgate (attrib.)</author>
+                        <title>Reson and Sensuallyte</title>
+                        <note>Freely translated from <title>Les Echecs amoureux</title> (lines 1–4873)</note>
```
Comment (holfordm) [inline collections/Fairfax/MS_Fairfax_16.xml:545]: This long text should probably have an authority record. Should some other items as well?

Comment (adunning) [reply collections/Fairfax/MS_Fairfax_16.xml:548]: It wouldn't be difficult to create IDs systematically for everything in DIMEV in a Bodleian MS – for another PR …

**Pull request merged**